src/templates/Article/base.js

```diff
@@ -231,6 +235,9 @@ const createBase = ({ metaBody, metaHeadlines }) => {
   const figureCaption = {
     matchMdast: matchParagraph,
     component: FigureCaption,
+    props: (node, index, parent, { ancestors }) => ({
+      groupCaption: parent.identifier === 'FIGUREGROUP'
```

Comment:
Wouldn't the parent of the caption be the FIGURE and only the parent of the parent FIGUREGROUP?

Reply:
not if it's a group caption

Reply:
`FIGURE`s in `FIGUREGROUP`s can have their own `FIGURECAPTION`s, but there are also shared `FIGURECAPTION`s, which are a direct descendant of `FIGUREGROUP`s and are the ones we are targetting here.

Reply:
make sense, sorry didn't get it earlier